### PR TITLE
Improve the FIDO registration and sign-in addition experience

### PR DESCRIPTION
**Pull request by thivi**
### Purpose
Improved the following flows:
1. FIDO2 Security Key registration flow in the My Account app.
2. Added a usernameless authentication template in the Console app.
3. The authentication flow in the authentication portal.

Resolves https://github.com/wso2/product-is/issues/13025

 image
image

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on final implementation.
- [ ] Documentation provided. (Add links if there's any)
- [ ] Unit tests provided. (Add links if there's any)
- [ ] Integration tests provided. (Add links if there's any)

### Related PRs
- Related PR `#1` or (None)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: modules/theme/src/themes/default/elements/divider.overrides [hunk 1]
```diff
@@ -11,4 +11,8 @@
         margin-top: @dividerCompactMargin;
         margin-bottom: @dividerCompactMargin;
     }
+
+    &.fido-list-divider {
```
Comment (brionmario) [inline modules/theme/src/themes/default/elements/divider.overrides:15]: Shall we make this genric? Divider already supports few variations `2x, 3x` etc.

File: modules/access-control/src/access-control-context-provider.tsx [hunk 2]
```diff
@@ -105,79 +105,79 @@ export const AccessControlContext: FunctionComponent<PropsWithChildren<AccessCon
                     get(featureConfig, "branding.scopes.delete"),
```
Comment (brionmario) [inline modules/access-control/src/access-control-context-provider.tsx:105]: Better if can do optional chaining here as well instead of loadsh get

**Review by brionmario — APPROVED** [review]

File: modules/access-control/src/access-control-context-provider.tsx (hunk 2 above)
Comment (thivi) [reply modules/access-control/src/access-control-context-provider.tsx]: Resolved by https://github.com/wso2/identity-apps/pull/2815/commits/0821946c8f541536e273de6495b2d9cf0daec665

File: modules/theme/src/themes/default/elements/divider.overrides (hunk 1 above)
Comment (thivi) [reply modules/theme/src/themes/default/elements/divider.overrides]: Resolved by https://github.com/wso2/identity-apps/pull/2815/commits/f90c3ca553a1f5b6bebde334933133841e096a00

**Review by brionmario — APPROVED** [review]

**Pull request merged**